examples/django/charm/charmcraft.yaml

```diff
@@ -80,6 +80,7 @@ requires:
     interface: loki_push_api
   postgresql:
     interface: postgresql_client
+    optional: False
```

Comment:
Is the `optional` parameter `None` by default, does this imply that it is falsy by default?

Reply:
It is initialised to False in ops by default if not set. We are aware that this will make some charms block....